relational/table.py

```diff
@@ -52,6 +54,48 @@ def nv_row(self, proj, name='', value='', data=''):
         return row
 
 
+def _get_mohcd_units(proj, source_override=False):
+    """
+    Gets net new units and bmr counts from the mohcd dataset.  Prioritizes
+    data from MOHCDPipeline, and falls back to MOHCDInclusionary if none
+    found.
+
+    If source_override is specified, will pull numbers only from the given
+    source.  Otherwise will pull from pipeline and inclusionary mohcd data.
+    Nothing technically prevents you from providing a non-mohcd source, but
```

Comment:
Super uber nit: Couldn't we just check that the source provided is in the sources we have in here to work with to protect against this?